### DISCUSSION TITLE
Unable to log into Kibana after updating to 2.3.90

### DISCUSSION THREAD

**Discussion by Hammer-Bear** [Security-Onion-Solutions/securityonion#6354] [Unsupported Versions]
So just upgraded our distributed SO environment to 2.3.90 and it looks like everything ran successfully apart from we are unable to log into Kibana/Elastic with any of our accounts including the default one set up from the original set-up. Its the first time we've ran an update via soup so I could be missing something but I can't see anything obvious.

**Comment by dougburks** [comment]
What version were you on before running `soup`?

Are there any errors in `/root/soup.log`?

What is the output of `sudo so-user-list`?

**Reply by Hammer-Bear** [reply]
the previous version was 2.3.61 MSEARCH hotfix, soup log doesn't show any errors and didn't prompt any when it completed. The user list is the default account we set as superuser and the other account which is mine also shown as superuser
![image](https://user-images.githubusercontent.com/36236670/143886055-9b4fb6d6-f8a1-4fc9-81f4-5774f1be714d.png)

**Reply by dougburks** [reply]
Have you tried resetting your password or creating a new account?
